modules/sdk-coin-bera/test/unit/bera.ts

```diff
@@ -1,28 +1,753 @@
-import { BitGoAPI } from '@bitgo/sdk-api';
+import * as should from 'should';
+import { bip32 } from '@bitgo/utxo-lib';
+import { common, FullySignedTransaction, TransactionType, Wallet } from '@bitgo/sdk-core';
+import * as secp256k1 from 'secp256k1';
+import nock from 'nock';
 import { TestBitGo, TestBitGoAPI } from '@bitgo/sdk-test';
-import { Tbera, Bera } from '../../src';
+import { BitGoAPI } from '@bitgo/sdk-api';
+import { OfflineVaultTxInfo, optionalDeps, SignTransactionOptions } from '@bitgo/abstract-eth';
+
+import { Bera, Tbera, TransactionBuilder, TransferBuilder } from '../../src';
+import * as mockData from '../fixtures/bera';
+import { getBuilder } from '../getBuilder';
+
+nock.enableNetConnect();
 
 describe('Bera', function () {
   let bitgo: TestBitGoAPI;
+  let basecoin;
+  let hopTxBitgoSignature;
+
+  const address1 = '0x174cfd823af8ce27ed0afee3fcf3c3ba259116be';
+  const address2 = '0x7e85bdc27c050e3905ebf4b8e634d9ad6edd0de6';
+  const hopContractAddress = '0x47ce7cc86efefef19f8fb516b11735d183da8635';
+  const hopDestinationAddress = '0x9c7e8ce6825bD48278B3Ab59228EE26f8BE7925b';
+  const hopTx =
+    '0xf86b808504a817c8ff8252ff949c7e8ce6825bd48278b3ab59228ee26f8be7925b87038d7ea4c68000801ca011bc22c664570133dfca4f08a0b8d02339cf467046d6a4152f04f368d0eaf99ea01d6dc5cf0c897c8d4c3e1df53d0d042784c424536a4cc5b802552b7d64fee8b5';
+  const hopTxid = '0x4af65143bc77da2b50f35b3d13cacb4db18f026bf84bc0743550bc57b9b53351';
+  const userReqSig =
```

Comment:
I think we don't need to add hop transactions related tests for new coins as hop transactions shouldn't be supported for new coins

Reply:
okay